### DISCUSSION TITLE
一部のキーの反応が悪い

### DISCUSSION THREAD

**Discussion by HogeTatu** [Yowkees/keyball#320] [Q&A] [answered]
星印を付けたキーの反応が悪い状態です。症状としてはキー押下で入力されないことが多く、一度強めに押し込むとしばらく入力が可能になるといった状態なので接触不良の可能性は高いと思います。キーソケットとキースイッチの交換は試し、ProMicroの左右入れ替えも行ってみたのですが改善しない状態です。他に試すべきことは何があるでしょうか？お手数ですが、情報頂けると助かります。

![keyball44_背面01](https://github.com/Yowkees/keyball/assets/1815549/01267fd7-c0fa-4ad4-916e-37a2b74345df)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。返信が遅くなり申し訳ありませんでした。

キーソケットはすでに交換済ということで、念のため下記画像のオレンジ色をつけたダイオードの半田付けを見直してみてください。

強く押すと基板がたわんで回路が変化するようですので、ミドル基板のパッドが剥がれかけている等の配線にダメージがあるのかも知れません。ダイオードの手直しで改善しないようでしたら、下記画像の同色どうしをエナメル線やゴム被覆銅線で接続していただくと改善できると思います。

お手数ですがご対応お願いいたします。

![KB44_1keyfix](https://github.com/Yowkees/keyball/assets/84209772/74f9e547-cd88-4a9f-8470-5424609edf53)

**Reply by HogeTatu** [reply]
オレンジ色の箇所のダイオードの半田付けをし直し、しばらく使用していますが安定したようです。ご対応ありがとうございました。

**Comment by Yowkees** [comment]
無事に直ったようでよかったです。安心しました。
ぜひ末永くご愛用いただければ幸いです。
